File: Praktikum 7/Python/PRAK604-2310817210004-Allano Lintang Ertantora.py
kalimatSatu = input()
kalimatDua = input()
if len(kalimatSatu) != len(kalimatDua):
    print('Panjang kalimat berbeda, pesan palsu')
else:
    sama = 0
    beda = 0
    for i in range(len(kalimatSatu)):
        if kalimatSatu[i] == kalimatDua[i] and kalimatSatu[i] != " " and kalimatDua[i] != " ":
            print("*", end="")
            sama += 1
        elif kalimatSatu[i] == ' ' and kalimatDua[i] == ' ':
            print(" ", end="")
        else:
            print("#", end="")
            beda += 1
    print()
    print(f'* = {sama}')
    print(f'# = {beda}')
    if sama > beda:
        print('Pesan asli')
    else:
        print('Pesan palsu')    

File: Praktikum 7/C/PRAK604-2310817210004-Allano-Lintang-Ertantora.c
#include <stdio.h>
#include <string.h>
int main(){
    char kalimatSatu[50], kalimatDua[50];
    fgets(kalimatSatu, sizeof(kalimatSatu), stdin);
    fgets(kalimatDua, sizeof(kalimatDua), stdin);
    if (strlen(kalimatSatu) != strlen(kalimatDua)){
        printf("Panjang kalimat berbeda, pesan palsu\n");
    }else{
        int sama = 0, beda = 0;
        for (int i = 0; i < strlen(kalimatSatu)-1; i++){
            if (kalimatSatu[i] == kalimatDua[i] && kalimatSatu[i] != ' ' && kalimatDua[i] != ' '){
                printf("*");
                sama++;
            }else if (kalimatSatu[i] == ' ' && kalimatDua[i] == ' '){
                printf(" ");
            }else{
                printf("#");
                beda++;
            }
        }printf("\n* = %d\n# = %d\n", sama, beda);
        if (sama > beda){
            printf("Pesan asli\n");
        }else{
            printf("Pesan palsu\n");
        }
    }
    return 0;
}